src/ott/solvers/linear/sinkhorn.py

```diff
@@ -329,6 +330,7 @@ class SinkhornOutput(NamedTuple):
   threshold: Optional[jnp.ndarray] = None
   converged: Optional[bool] = None
   inner_iterations: Optional[int] = None
+  n_iters: int = -1
```

Comment:
I think it can be safely removed, was not being used anywhere and I don't think it helps when looking at the `errors`. Same goes for `threshold`, would also remove.
Will try in the upcoming weeks to unify the output interface for our problems.

Reply:
Or was/is there a reason why `threshold` was put there before? Maybe because of traceability?

Reply:
The `threshold` is there to settle `converged` in the case where `min_iterations==max_iterations`. It was added specifically for that case.

For `inner_iterations`, i think we should have the user in mind. If the user saves a `SinkhornOutput` as a checkpoint, it's impossible to reconstruct the "true" number of iterations that were needed to reach a certain error level, nor the `threshold` that conditioned stopping (apart maybe now by looking at last error not marked as -1, and then divide now `n_iters` by that number, but that does sound very clean)